docs/csharp/language-reference/compiler-options/doc-compiler-option.md

```diff
@@ -29,7 +29,7 @@ The **-doc** option allows you to place documentation comments in an XML file.
 ## Remarks  
  In source code files, documentation comments that precede the following can be processed and added to the XML file:  
   
-- Such user-defined types as a [class](../keywords/class.md), [delegate](../keywords/delegate.md), or [interface](../keywords/interface.md)  
+- Such user-defined types as a [class](../keywords/class.md), [delegate](../builtin-types/reference-types.md), or [interface](../keywords/interface.md)  
```

Comment:
same here. all these redirects for object, string, delegate should have the anchor added.

```suggestion
- Such user-defined types as a [class](../keywords/class.md), [delegate](../builtin-types/reference-types.md#the-delegate-type), or [interface](../keywords/interface.md)  
```